gensrc/thrift/RuntimeFilter.thrift

```diff
@@ -29,8 +29,10 @@ enum TRuntimeFilterBuildJoinMode {
   NONE,
   BORADCAST,
   PARTITIONED,
-  BUCKET_SHUFFLE,
-  COLOCATE
+  LOCAL_HASH_BUCKET,
```

Comment:
Is there a grayscale upgrade problem here ?

Reply:
it is ok, just change the symbolic names.